src/status_im2/common/home/top_nav/view.cljs

```diff
@@ -0,0 +1,59 @@
+(ns status-im2.common.home.top-nav.view
+  (:require
+    [quo2.core :as quo]
+    [react-native.core :as rn]
+    [status-im.multiaccounts.core :as multiaccounts]
+    [status-im2.common.home.top-nav.style :as style]
+    [status-im2.constants :as constants]
+    [utils.debounce :refer [dispatch-and-chill]]
+    [utils.re-frame :as rf]))
+
+(defn connectivity-sheet
+  []
+  (let [peers-count  (rf/sub [:peers-count])
+        network-type (rf/sub [:network/type])]
+    [rn/view
```

Comment:
Can this be a fragment?

Reply:
I don't think so as there is absolute elements within it that depend on this (afaik)